modAionImpl/src/org/aion/zero/impl/AionBlockchainImpl.java

```diff
@@ -1212,7 +1220,14 @@ private AionBlockSummary applyBlock(IAionBlock block) {
                         block.getNrgLimit(),
                         LOGGER_VM,
                         getPostExecutionWorkForApplyBlock());
-        List<AionTxExecSummary> executionSummaries = executor.execute();
+
+        List<AionTxExecSummary> executionSummaries = null;
+        try {
+            executionSummaries = executor.execute();
+        } catch (VMException e) {
+            LOGGER_VM.error(e.toString());
```

Comment:
similarly, needs the message and stack trace

Reply:
updated